TreeMaker/python/doLostLeptonBkg.py

```diff
@@ -8,19 +8,38 @@ def doLostLeptonBkg(self,process,METTag):
             pfCandsTag  = cms.InputTag('packedPFCandidates')
         )
 
+    from TreeMaker.TreeMaker.TMEras import TMeras
     from TreeMaker.Utils.isolationproducer_cfi import isolationproducer
-    process.IDMuonMiniIso = isolationproducer.clone(
+    process.IDMuonMiniIso = isolationproducer.clone()
+    TMeras.TM2016.toModify(process.IDMuonMiniIso,

```

Comment:
i would keep the common values in the `clone()` statement so they do not need to be copied, and then only put 2016- or 2017-specific changes in the `toModify()` statements. (throughout this file and the next one)

Reply:
Between 2016 and 2017 within a lepton category, yes. But since both the electrons and the muons use the same producer and the values between 2016(muon) and 2016(electron) are different I needed to specify something. The value in the cfi file is generic as to not specify either electron or muon.

Reply:
yes, so put it once in the `clone()` statement (like it was before) instead of putting it twice in the `toModify()` statements